bundle/src/main/java/com/adobe/acs/commons/email/impl/EmailServiceImpl.java

```diff
@@ -123,6 +136,72 @@
         return failureList;
     }
 
+    @Override
+    public List<InternetAddress> sendEmail(String templatePath, Map<String, String> emailParams, Map<String, DataSource> attachments, InternetAddress... recipients) {

```

Comment:
There any reason the _sendEmail_ method w/o attachments can't call this with an empty map? The method appears to already handle that case, and it would consolidate all of the logic to a single(-ish) point.

Thoughts?